src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -95,7 +95,7 @@ private void determineCollectionStatus(DarCollectionSummary summary,
     //if there are some elections open, status is in process
     //if all elections are closed or canceled and electionCount == datasetCount, status is complete
     if (electionCount.equals(0)) {
-      summary.setStatus(DarCollectionStatus.UNREVIEWED.getValue());
+      summary.setStatus(DarCollectionStatus.SUBMITTED.getValue());
```

Comment:
If all the uses of this enum are as the string constant, does it need to be an enum? Either `DarCollectionSummary` should be type `DarCollectionStatus` instead of `String`, or these enums could be replaced with `static final String` constants.

Reply:
These certainly could be strings instead. There is future work to consolidate the state machine logic around status and actions, so we can probably tackle this there.